csharp/source/autowrap/csharp/common.d

```diff
@@ -0,0 +1,58 @@
+module autowrap.csharp.common;
+
+package template ArrayElementType(T : T[])
+{
+    alias T ArrayElementType;
```

Comment:
Please don't use old-style aliases, they're a lot harder to read.

Reply:
Dead code, removed.